src/lib/ares_cancel.c

```diff
@@ -37,17 +37,16 @@
  */
 void ares_cancel(ares_channel channel)
 {
-  if (ares__llist_len(channel->all_queries) > 0)
-  {
+  if (ares__llist_len(channel->all_queries) > 0) {
     ares__llist_node_t *node = NULL;
     ares__llist_node_t *next = NULL;
 
     /* Swap list heads, so that only those queries which were present on entry
      * into this function are cancelled. New queries added by callbacks of
      * queries being cancelled will not be cancelled themselves.
      */
-    ares__llist_t *list_copy = channel->all_queries;
-    channel->all_queries = ares__llist_create(NULL);
+    ares__llist_t      *list_copy = channel->all_queries;
+    channel->all_queries          = ares__llist_create(NULL);
```

Comment:
Wait, why is this padding the code with spaces here? Is this just because it makes an assign below the variable declaration with an assign?

Reply:
yeah, looks like it. I think this is because there isn't a newline between the variable declaration and the assignment on the next line, as it won't do that across newlines.  

There are definitely some areas where its not flexible enough.